docs/sources/developers/kinds/composable/testdatadataquery/schema-reference.md

```diff
@@ -0,0 +1,120 @@
+---
+keywords:
+  - grafana
+  - schema
+title: TestDataDataQuery kind
```

Comment:
This looks funky :smile:

Reply:
yeah that's a yuck, for sure. The way we name composable kinds in different codegen contexts is a larger item for discussion, one of the things i handwaved over to be addressed later with the unification PR last week